billing-service/src/test/java/com/petclinic/billing/exceptions/InvalidInputExceptionTest.java

```diff
@@ -30,4 +30,12 @@ void TestInvalidInputMessageAndCauseConstructor() {
             throw new InvalidInputException("Something went wrong", new Exception());
         });
     }
+
+
+    @Test
+    void TestInvalidInputMessageOnly() {
+        assertThrows(InvalidInputException.class, () -> {
+            throw new InvalidInputException("Something went wrong");
```

Comment:
Both input exception and not found exception return the same error message "Something went wrong", do you think we should give specific messages to each exception